types/promotion.ts

```diff
@@ -0,0 +1,18 @@
+export interface Promotion {
+  id: string;
+  conferenceId: string;
+  registrantTypeIds: string[];
```

Comment:
I think the preferred way to write `string[]` is like the below? Although, I so like the ease of `string[]`
```suggestion
registrantTypeIds: Array<string>;```